store/json.go

```diff
@@ -82,7 +83,7 @@ func (kvs *jsonFileStore) Read(key string, value interface{}) error {
 		}
 
 		if len(b) == 0 {
-			log.Printf("Unable to read file %s, was empty", kvs.fileName)
+			log.Logger.Info("Unable to read was empty", zap.String("file", kvs.fileName))
```

Comment:
- This sentence reads a bit odd. Should just be `File is empty`.
- Should be logged as an error. Does `Logger` have an `Error` method?